packages/netlify-cms-core/src/components/Editor/EditorControlPane/EditorControl.js

```diff
@@ -330,7 +331,8 @@ class EditorControl extends React.Component {
             {fieldHint && (
               <ControlHint active={isSelected || this.state.styleActive} error={hasErrors}>
                 <ReactMarkdown
-                  allowedElements={['a', 'strong', 'em']}
+                  plugins={[gfm]}
```

Comment:
Per the [docs](https://github.com/remarkjs/react-markdown/tree/ce6c1a71c17280e753e54e919511cd8bafadf86e#use) this should be `remarkPlugins={[gfm]}`